### DISCUSSION TITLE
License appearing on a GitHub Project

### DISCUSSION THREAD

**Discussion by eliesaikali** [community/community#56919] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Dear all,

On my github project, I see in the License section (in the right section of the page) 2 licenses for my code : 
 **Unknown**, **BSD-3-Clause licenses found**.

However, in my project I only have a license.txt file, which is a BSD license... So no idea why it shows Unknown ....

Is it possible to modify this and keep only BSD-3-Clause license ? 

Thanks for your help 

ES

**Comment by DSDmark** [comment]
Hi @eliesaikali,

Can you **specific more** about the details? For better understanding 😅. Also, you can add some **screenshots** which you were facing 👍 and also you can add repo link.

**Reply by eliesaikali** [reply]
Hello @DSDmark 

Here is the link : https://github.com/cea-trust-platform/trust-code

As you see, it is showing 2 licenses : Unknown, BSD-3-Clause licenses found

The question is how to remove the "Unkown" licence found ^^

Its more clear now ?

**Reply by airtower-luna** [reply]
The UI says the unknown license is in a file called `license`, which obviously doesn't exist. I can see in the history that it was [renamed somewhat recently](https://github.com/cea-trust-platform/trust-code/commit/661b0f6dbf327086e4f84df41337451443349f64), so I assume it's a caching issue or something like that. Was that commit pushed only recently? If yes I'd just wait a bit, if not you might want to report a bug to GitHub support. :thinking:

**Reply by eliesaikali** [reply]
yes it a recent commit ... thanks for your answer @airtower-luna , i will wait then :) 
Cheers

**Reply by DSDmark** [reply]
@eliesaikali,
I **forked** your repo to see the **license**, it looks like this 

![Screenshot from 2023-06-01 18-09-04](https://github.com/community/community/assets/85744816/eb4e0a40-226c-4a94-86bd-5ffe8a67b889)

So, As **airtower-luna** said, it may be a caching issue. 

You could try to rename it **license.txt** to **license** only, it's working on fork repo

![Screenshot from 2023-06-01 18-07-48](https://github.com/community/community/assets/85744816/fac9944c-1b79-4b09-8e66-6270f5a5c098)

If all things not working for you, then wait, it may resolve. Or, You can create a ticket:- [**GitHub Support**](https://support.github.com/request)
